tests/perf/actor_id_scale/test.js

```diff
@@ -0,0 +1,55 @@
+import http from "k6/http";
```

Comment:
Should this file be included with `go:embed`? Unless you add a custom Dockerfile, I believe we just compile the Go app and don't have a way to include other files in the container. Unless I'm wrong

Reply:
Thanks for introducing me the `go:embed`, very interesting.

But I don't think we do need to embed the test file because it's being used by the test files only, so no dockerfile is needed,

Even considering that we don't need a dockerfile/compile go code, I have to change the k6 runner from receiving a script name to (probably) receive a bytes array - because the k6.go code is the one that read the file content which is dynamic. I feel that this "optimization" is unnecessary for the usecase of running tests, what do you think?

Reply:
Oh you’re right. I thought this was a test app but instead it’s a test case, so you’re right there’s no need for embeds.

Don’t worry about optimizations in test cases :)